compare_platforms/compare_platforms_stacksmash.c

```diff
@@ -0,0 +1,109 @@
+/*
+ * This example is meant to be run in multiple instances launched simultaneously.
+ * It can also be run as a single instance.
+ *
+ * Usage:
+ * prompt> ./compare_platforms_stacksmash & ./compare_platforms_stacksmash &
```

Comment:
I must admit that I don't understand why this needs two instances running side by side to be a problem -- this process doesn't seem to interact with the outside world in such a way that would cause a problem? Perhaps I'm missing something!

Reply:
Yes, I think the multiple processes thing is a red herring, so it's best removed.

Reply:
Fixed in 632acd64b8eee4b438b2b4bc36faa5507f61eb54